### ISSUE TITLE
Player screen crashes when adding combatant

### ISSUE THREAD

**Issue by dmmike** [cynicaloptimist/improved-initiative#378]
It seems that when you add a combatant to a battle while you have round tracking turned on, the player screen goes haywire, causing it to fail in updating (or starting a new screen, leaving it empty). This is a theory as the debug console shows an error regarding the round counter.

Turning the option off in the settings fixes the issue and has the player screen working again.

**Comment by cynicaloptimist** [comment]
Excellent find, thanks for the report! This probably explains a lot of reports of the flaky player view that I've been seeing.

**Comment by cynicaloptimist** [comment]
I can't reproduce this bug. Here is what I tried:

1. Turn on "Display Round Counter" setting
2. Launch Player View
3. Open dev tools in player view
4. Add an Acolyte statblock to the encounter

Can you give more specific repro steps? Is there a particular statblock or character that this happens for, and if so can you share the JSON?

Thanks!

**Comment by dmmike** [comment]
Damn, I honestly thought that was the cause!
I can't seem to reproduce this at the moment either, I'll enable the options to true again and I'll run into it eventually, I'll be sure to write down the situation I was in at that moment.

I'm still absolutely sure it is tied to the round counter, by the way, as that was the variable that was in the error log in the console, and the problem was fixed as soon as I disabled that option.

I'll try and mess around, see if I can recreate the problem!
